src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -255,7 +255,7 @@ public function testNullsafeCompile($expression, $foo)
         $this->assertNull(eval(sprintf('return %s;', $expressionLanguage->compile($expression, ['foo' => 'foo']))));
     }
 
-    public function provideNullsafe()
+    public function provideNullSafe()
```

Comment:
while at it, there are two more occurrences of `Nullsafe` in the file :)

Reply:
Fixed